internal/venice-common/src/main/java/com/linkedin/venice/system/store/MetaStoreWriter.java

```diff
@@ -360,7 +368,16 @@ Map<String, VeniceWriter> getMetaStoreWriterMap() {
 
   VeniceWriter getOrCreateMetaStoreWriter(String metaStoreName) {
     return metaStoreWriterMap.computeIfAbsent(metaStoreName, k -> {
-      PubSubTopic rtTopic = pubSubTopicRepository.getTopic(Utils.composeRealTimeTopic(metaStoreName));
+      Store store = storeResolver.apply(metaStoreName);
+      int largestUsedRTVersionNumber;
+      VeniceSystemStoreType type = VeniceSystemStoreType.getSystemStoreType(store.getName());
+      if (type != null && store.isSystemStore()) {
```

Comment:
If the store is not system store, should we just return null, as this function is for system store only.